scripts/setup-local-cluster.sh

```diff
@@ -102,6 +103,8 @@ function cleanup {
     lsof -i ":${port}" -s TCP:LISTEN -t | xargs -I {} -n 1 kill {}
   done
 
+  rm ${env_file}
```

Comment:
What if `env_file` does not exist?

Reply:
The command will fail, but the script will keep running.

Reply:
Yeah, I can add existence check to avoid errors in the output, but in general should not matter.